link/kprobe.go

```diff
@@ -49,6 +49,8 @@ type KprobeOptions struct {
 	//
 	// Needs kernel 5.15+.
 	Cookie uint64
+	// Kprobe symbol offset, or uprobe path offset.
```

Comment:
this is not the uprobe offset as this options are only used in the context of kprobes

PS. don't close and open PRs for small changes, just add a commit and optionally rebase/squash

Reply:
ok, thanks! I am unfamiliar with github PR.

Reply:
👍🏻